src/ngraph/ngraph_emitter.cc

```diff
@@ -62,17 +62,30 @@ void Emitter::ClearOpMap() {
  * based index), where
  * negative values means indexing from the right.
  */
+inline size_t transform_axis(int axis, int shape_size) {
+  assert(abs(axis) <= shape_size);
+  // convert negative axis index to postive (counting from right per mxnet
+  // convention)
+  return axis < 0 ? shape_size + axis : axis;
+}
+
 inline size_t get_default_transformed_axis(const NodePtr& node,
                                            const std::string& key,
                                            const int default_val,
                                            const int shape_size) {
-  int axis = get_default(node, key, default_val);
-  assert(abs(axis) <= shape_size);
-  // convert negative axis index to postive (counting from right per mxnet
-  // convention)
-  size_t transformed_axis = axis < 0 ? shape_size + axis : axis;
+  return transform_axis(get_default(node, key, default_val), shape_size);
+}
 
-  return transformed_axis;
+inline std::vector<size_t> get_default_transformed_axis(
+    const NodePtr& node, const std::string& key,
+    const ngraph::AxisVector& default_val, const ngraph::Shape shape) {
+  std::vector<size_t> axes;
+  assert(default_val.size() == shape.size());
+  for (size_t i = 0; i < default_val.size(); ++i) {
+    axes.push_back(transform_axis(
+        get_default(node, key, static_cast<int>(default_val[i])), shape[i]));
```

Comment:
I don't understand this line. We are expecting "axis" as a vector, why this is called in a loop?